templates/taxi/confirm_delete.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  {% if object is manufacturer %}

```

Comment:
Using `is` for object type checking might not work as expected in Django templates. Consider using `{% if object|instanceof:'Manufacturer' %}` or `{% if object|instanceof:'Car' %}` with a custom template filter to check the object's type.